smithy-model/src/main/java/software/amazon/smithy/model/loader/ModelAssembler.java

```diff
@@ -499,6 +517,9 @@ private LoaderVisitor createLoaderVisitor() {
 
         shapes.forEach(visitor::onShape);
         metadata.forEach(visitor::onMetadata);
+        pendingTraits.forEach(pendingTrait -> {
```

Comment:
Nit, personal taste -- I know this comes right after some `forEach` calls, but I prefer a normal for loop when possible.

Reply:
Fixed.